lib/setopt.c

```diff
@@ -1317,7 +1282,7 @@ CURLcode Curl_vsetopt(struct Curl_easy *data, CURLoption option, va_list param)
     /*
      * pass CURLU to set URL
```

Comment:
wrong, but also seems to point to the wrong line number

Reply:
If you'd like we can filter out messages of type `UNINITIALIZED_VALUE`.  Let me know and I'll make a PR.

Reply:
But this isn't an actual `UNINITIALIZED_VALUE`. The variable *is* initialized for this option, the analyzer is just deeply confused by this kind of code.

I'll do another iteration of the cleanup anyway since lots of analyzers will (wrongly) complain on this, I've been down this road before.

Reply:
With respect to the incorrect line numbers, what is happening seems clear.  It is related to analyze/push/comment racing and we are working it now.  Sorry this got in your way, I suspect this was unearthed now because you push far faster and more frequently than other users.

Reply:
Happy to serve as a stress test! :grin: